app/scripts/modules/core/src/account/account.service.ts

```diff
@@ -148,10 +153,7 @@ export class AccountService {
   }
 
   public listAllAccounts(provider: string = null, providerVersion: string = null): IPromise<IAccountDetails[]> {
-    return this.API.one('credentials')
-      .useCache()
-      .withParams({ expand: true })
-      .get()
+    return this.$q.when(this.accounts$.toPromise())
```

Comment:
semicolons make something something. (we really should have a lint rule for this...)

Reply:
they sure do make something, that's for sure.


wait


is it Java?? does it make Java??

Reply:
haha, damn split view diff confusing me. jokes on me.